episode-6.msrt

```diff
@@ -387,7 +391,8 @@
 78
 00:05:06,020 --> 00:05:10,110
 [ru] [саркастически] Эх, радуйся, Кэррот. Похоже, никто не увидит твое чУдное зелье..
-[en] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
+[en] [Version 1:] [sarcastically] Ah, be glad Carrot. It seems nobody will see your wondrous potion...
```

Comment:
I'd stick with "You're lucky" as it shows that Carrot is lucky that his mess-up won't be immediately discovered (save for the threat of having it dumped on the unsuspecting bird). Plus it leads into the next few scenes of "Hey, maybe this isn't a complete mess-up after all".

Plus, an English-speaking actress can have some fun with "you're lucky, Carrot". "It's your lucky day" will sound like a carnival barker when spoken.